### PR TITLE
Updated IssueVerified (#19)

### PR DESCRIPTION
**Pull request by Brett-Knecht**
Removed Data.value and added Successful, Failed, and Inconclusive links.

### CODE REVIEW

File: examples/events/EiffelIssueVerifiedEvent/simple.json [hunk 1]
```diff
@@ -36,6 +36,18 @@
     {
       "type": "ISSUE",
       "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee5"
```
Comment (jaden-young) [inline examples/events/EiffelIssueVerifiedEvent/simple.json]: Should remove link of type "ISSUE" from the example as it isn't a valid link type after changes

File: eiffel-vocabulary/EiffelIssueVerifiedEvent.md [hunk 2]
```diff
@@ -35,6 +27,24 @@ __Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDef
...
 __Description:__ Identifies an issue that this event pertains to.
 
+### SUCCESFUL_ISSUE
+__Required:__ Yes  
+__Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies an issue that this event pertains to.
+
+### FAILED_ISSUE
+__Required:__ Yes
```
Comment (jaden-young) [inline eiffel-vocabulary/EiffelIssueVerifiedEvent.md]: We should only require that one of `SUCCESSFUL_ISSUE`, `FAILED_ISSUE`, `INCONCLUSIVE_ISSUE` be required. So for each of these, mark them as `__Required:__ No` and make a note in the description at the top of the file.

**Review by jaden-young — CHANGES_REQUESTED** [review]

File: eiffel-vocabulary/EiffelIssueVerifiedEvent.md [hunk 3]
```diff
@@ -16,33 +16,27 @@
 --->
 
 # EiffelIssueVerifiedEvent (IV)
-The EiffelIssueVerifiedEvent declares that an issue, typically a requirement, has been verified by some means. It is different from [EiffelTestCaseFinishedEvent](./EiffelTestCaseFinishedEvent.md) in that multiple test case executions may serve as the basis for a single verification or, conversely, multiple issues may be verified based on a single test case execution.
+The EiffelIssueVerifiedEvent declares that an issue, typically a requirement, has been verified by some means. It is different from [EiffelTestCaseFinishedEvent](./EiffelTestCaseFinishedEvent.md) in that multiple test case executions may serve as the basis for a single verification or, conversely, multiple issues may be verified based on a single test case execution. Issues may only maintain one link of type ISSUE at a time.
```
Comment (jaden-young) [inline eiffel-vocabulary/EiffelIssueVerifiedEvent.md]: We don't want to say that `IssueVerified` can only reference one issue. In this paragraph we mention that "multiple issues may be verified based on a single test case execution." What we do want to say is something along the lines of "At least one link to an issue must be present, whether it be `"SUCCESSFUL_ISSUE"`, `"FAILED_ISSUE"`, or `"INCONCLUSIVE_ISSUE"`.

File: examples/events/EiffelIssueVerifiedEvent/simple.json (hunk 1 above)
Comment (Brett-Knecht) [reply examples/events/EiffelIssueVerifiedEvent/simple.json]: Makes Sense, Done

File: eiffel-vocabulary/EiffelIssueVerifiedEvent.md [hunk 4]
```diff
@@ -16,33 +16,27 @@
...
-__Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
-__Multiple allowed:__ Yes  
-__Description:__ Identifies an issue that this event pertains to.
-
 ### SUCCESFUL_ISSUE
 __Required:__ Yes  
 __Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
-__Multiple allowed:__ Yes  
+__Multiple allowed:__ No  
 __Description:__ Identifies an issue that this event pertains to.
```
Comment (jaden-young) [inline eiffel-vocabulary/EiffelIssueVerifiedEvent.md]: Should have a more descriptive description.

File: eiffel-vocabulary/EiffelIssueVerifiedEvent.md [hunk 5]
```diff
@@ -16,33 +16,27 @@
...
 
 ### FAILED_ISSUE
 __Required:__ Yes  
 __Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
-__Multiple allowed:__ Yes  
+__Multiple allowed:__ No  
 __Description:__ Identifies an issue that this event pertains to.
```
Comment (jaden-young) [inline eiffel-vocabulary/EiffelIssueVerifiedEvent.md]: Should have a more descriptive description.

File: eiffel-vocabulary/EiffelIssueVerifiedEvent.md [hunk 6]
```diff
@@ -16,33 +16,27 @@
...
 
 ### INCONCLUSIVE_ISSUE
 __Required:__ Yes  
 __Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
-__Multiple allowed:__ Yes  
+__Multiple allowed:__ No  
 __Description:__ Identifies an issue that this event pertains to.
```
Comment (jaden-young) [inline eiffel-vocabulary/EiffelIssueVerifiedEvent.md]: Should have a more descriptive description.

**Review by jaden-young — CHANGES_REQUESTED** [review]

**Comment by jaden-young** [comment]
Working on issue #19

**Review by jaden-young — APPROVED** [review]

**Pull request merged**